src/factorizations/svd.jl

```diff
@@ -34,33 +42,29 @@ function MatrixAlgebraKit.initialize_output(
   bm, bn = blocksize(A)
   bmn = min(bm, bn)
 
-  brows = blocklengths(axes(A, 1))
-  bcols = blocklengths(axes(A, 2))
-  slengths = Vector{Int}(undef, bmn)
+  brows = eachblockaxis(axes(A, 1))
+  bcols = eachblockaxis(axes(A, 2))
+  s_axes = Vector{eltype(brows)}(undef, bmn)
```

Comment:
```suggestion
  s_axes = similar(brows, bmn)
```
Probably doesn't matter, but this might be slightly more general and easier to read